tests/test.rb

```diff
@@ -44,11 +44,23 @@ def test_mailer_redirects_to_error_without_valid_configured_account
     post '/', { public_token: 'no-account',
                 name: 'butt',
                 email: 'butt@tp.com',
-                textarea: 'hi smelly',
+                verify: '',
                 action: 'value' }
 
     assert last_response.redirect?
     follow_redirect!
     assert_includes last_response.body, 'Account Not Configured'
   end
+
+  def test_mailer_redirects_to_err_bot_if_verify_field_is_not_empty
+    post '/', { public_token: 'default',
+                name: 'butt',
+                email: 'butt@tp.com',
+                verify: 'ima bot bot',
+                action: 'value' }
+
+    assert last_response.redirect?
+    follow_redirect!
+    assert_includes last_response.body, 'ERR! botnope'
```

Comment:
we don't actually want to return a success or fail response. We don't want the bot or someone looking at the logs to know that we blocked them

Reply:
I changed it to redirect to root/index